modules/bridge/src/main/java/eu/cloudnetservice/modules/bridge/platform/bungeecord/BungeeCordPlayerManagementListener.java

```diff
@@ -118,20 +118,21 @@ public void handle(@NonNull ServerConnectEvent event) {
   @EventHandler(priority = EventPriority.LOWEST)
   public void handle(@NonNull ServerKickEvent event) {
     if (event.getPlayer().isConnected()) {
+      if (event.getState() == ServerKickEvent.State.CONNECTING && event.getPlayer().getServer() != null) {
```

Comment:
Please keep the code consistent with the velocity one:
 * Extract the current server into a variable
 * Please add a comment why we're doing this
 * Please add a blank line after the if block